specification/resources/resource-manager/Microsoft.Resources/stable/2019-10-01/resources.json

```diff
@@ -4003,6 +4003,215 @@
           }
         }
       }
+    },
+    "/{scope}/providers/Microsoft.Resources/tags/default": {
+      "put": {
+        "tags": [
+          "Tags"
+        ],
+        "operationId": "Tags_ResourceCreate",
+        "description": "Create or Replace existing tags with pass-in tags.",
+        "parameters": [
+          {
+            "name": "scope",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The scope of the resources.",
+            "x-ms-skip-url-encoding": true
+          },
+          {
+            "name": "api-version",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "The API version to use for the operation."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/TagRequest"
+            },
+            "description": "Parameters for creating multiple tags."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns added tag information about the resource.",
+            "schema": {
+              "$ref": "#/definitions/TagResponse"
+            }
+          },
+          "400": {
```

Comment:
Do you need to also call out the schema for the 400 response? (Not sure what the answer is here).

Reply:
remove the 400 response from here

Reply:
and the 404 as well

Reply:
> and the 404 as well

Could you please explain a little more about removing 404? thanks

Reply:
error responses are not described explicitly in swagger. the SDK will generate a separate class for each response. SO we typically do it only for success responses. All error responses are captured via the "default" response